e2e_test/source/basic/kafka.slt

```diff
@@ -15,9 +15,20 @@ create materialized source s1 (v1 int, v2 varchar) with (
   scan.startup.mode = 'earliest'
 ) row format json
 
+statement ok
+create source ps1 (v1 int, v2 varchar) with (
+  connector = 'kafka',
+  topic = 'kafka_1_partition_topic',
+  properties.bootstrap.server = '127.0.0.1:29092',
+  scan.startup.mode = 'earliest'
+) row format json
```

Comment:
please consider moving the batch-related test cases to another slt file. There has already been too many tests here

Reply:
+1, I also feel difficult to read the tests.